clang/lib/DPCT/Utility.cpp

```diff
@@ -267,6 +267,9 @@ SourceProcessType GetSourceFileType(const clang::tooling::UnifiedPath &SourcePat
   if (Extension == ".cuh") {
     return SPT_CudaHeader;
   }
+  if (Extension == ".c") {
```

Comment:
same as above.

Reply:
The .C(capital letter) usually means C++ source file extension.
https://gcc.gnu.org/onlinedocs/gcc/Invoking-G_002b_002b.html